src/core/ComponentCell/style.ts

```diff
@@ -0,0 +1,31 @@
+import { styled } from "@mui/material/styles";
+import { CommonThemeProps, getSpaces } from "../styles";
+
+const contentPositionMapping = {
+  center: "center",
+  left: "flex-start",
+  right: "flex-end",
+};
+export interface ComponentCellExtraProps extends CommonThemeProps {
+  contentPosition?: "left" | "center" | "right";
+}
+
+export const StyledTableData = styled("td")`
```

Comment:
BasicCell component has the same export. Can you change `StyledTableData` to something else?